BiasAndFairnessServers/src/utils/handle_files_uploads.py

```
@@ -0,0 +1,79 @@
+import gzip
+import typing
+from fastapi import FastAPI, UploadFile
+from crud.bias_and_fairness import upload_model, upload_data, insert_metrics
+from utils.run_bias_and_fairness_check import analyze_fairness
+from database.db import get_db
+from database.redis import set_job_status
+
+app = FastAPI()
+
+@typing.no_type_check
+async def process_files(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)